.swcrc.build

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
I'm worried about this and .swcrc getting out of sync, but I don't touch these files often so it might be fine. There's also the option of writing a script to generate these files that I run the few times I actually do touch this - that could also be fine.

Reply:
I scripted these out, it's fine.